src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+
+      <div class="bem-outsideblock1">
+        <div class="picture-bem1">
+          <img class="picture-bem1__elem"
+            src="images/imac.jpeg"
+            alt="Product's picture"
+        >
+        </div>
+      </div>
+
+      <div class="card-text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="code-text">Product code: 195434</div>
+
+      <div class="star">
```

Comment:
```suggestion
      <div class="card__review">
```

OR `card__rate` `card__review-container`